cumulus/polkadot-omni-node/README.md

```diff
@@ -0,0 +1,64 @@
+# Polkadot Omni Node
+
+This is a white labeled implementation based on [`polkadot-omni-node-lib`](https://crates.io/crates/polkadot-omni-node-lib).
+It can be used to start a parachain node from a provided chain spec file. It is only compatible with runtimes that use block
+number `u32` and `Aura` consensus.
+
+## Installation
+
+Download & expose it via `PATH`:
+
+```bash
+# Download and set it on PATH.
+wget https://github.com/paritytech/polkadot-sdk/releases/download/<stable_release_tag>/polkadot-omni-node
+chmod +x polkadot-omni-node
+export PATH="$PATH:`pwd`"
+```
+
+Compile & install via `cargo`:
+
+```bash
+# Assuming ~/.cargo/bin is on the PATH
+cargo install polkadot-omni-node
+```
+
+## Usage
+
+A basic example for an Omni Node run starts from a runtime which implements the [sp_genesis_builder::GenesisBuilder](https://docs.rs/sp-genesis-builder/latest/sp_genesis_builder/trait.GenesisBuilder.html).
+The interface mandates the runtime to expose a [named-preset](https://docs.rs/staging-chain-spec-builder/6.0.0/staging_chain_spec_builder/#generate-chain-spec-using-runtime-provided-genesis-config-preset).
```

Comment:
```suggestion
The interface mandates the runtime to expose a [named-preset](https://docs.rs/staging-chain-spec-builder/latest/staging_chain_spec_builder/#generate-chain-spec-using-runtime-provided-genesis-config-preset).
```

We should be consistent in using `latest`, I suppose?

Reply:
[62f757c](https://github.com/paritytech/polkadot-sdk/pull/5954/commits/62f757c2c0920bb95b93fcd73e700112a33d7eeb)